packages/api/endpoints/granules.js

```diff
@@ -243,9 +243,13 @@ const putGranule = async (req, res) => {
     log.error('failed to update granule', error);
     return res.boom.badRequest(errorify(error));
   }
-  return _returnPutGranuleStatus(isNewRecord, apiGranule, res);
+  return _returnPatchGranuleStatus(isNewRecord, apiGranule, res);
 };
 
+function put(req, res) {
+  return res.boom.badRequest('put method not implemented');
```

Comment:
Should we add that we _do_ have a PATCH method? e.g.

>PUT method not implemented. Use PATCH to update a Granule.

Or similar.

Reply:
Also we're not routing to `put`? 🤔. I guess that makes sense as we don't want to completely break existing update calls.